[FIX] 빌드에러 해결

## 🚨 Issue Description

> 어떤 상황에서 버그가 발생하는지 설명해주세요.

- Type error: Cannot find module '@/app/lounge/_component/LoungeClient' or its corresponding type declarations.

## 🛠️ Proposed Solution

> 어떻게 수정할 계획인지 적어주세요.

-

## 🧪 Verification Steps

- [ ] 수정 후 정상 동작 확인

--- a/src/app/lounge/page.tsx
+++ b/src/app/lounge/page.tsx
@@ -2,7 +2,6 @@ import { Metadata } from "next";
 import Link from "next/link";
 import { BtnCommon } from "@/components/ui/BtnCommon";
 import HotPostList from "@/app/lounge/_component/HotPostList";
-import LoungeClient from "@/app/lounge/_component/LoungeClient";
 import PrefetchBoundary from "@/components/boundary/PrefetchBoundary";
 import { Suspense } from "react";
 import LoungeSkeleton from "@/components/skeleton/LoungeSkeleton";

